app/controllers/spam2_controller.rb

```diff
@@ -1,18 +1,18 @@
 class Spam2Controller < ApplicationController
-  before_action :require_user, only: %i(spam spam_revisions mark_comment_spam publish_comment spam_comments)
+  before_action :require_user, only: %i(_spam _spam_revisions _spam_comments)
 
   def _spam
```

Comment:
Hey @keshavsethi , not a big deal but just wanted to know why you choose to use ` _methodname`

Reply:
spam is method name for /spam(admin controller) while "_spam" is method name for /spam2 (spam2 controller). There was an error in method name so I corrected it

Reply:
gotcha :+1: